includes/class-attachments.php

```diff
@@ -933,4 +961,59 @@ private static function get_files_gallery_block( $files ) {
 
 		return $gallery;
 	}
+
+	/**
+	 * Save a remote actor's avatar locally.
+	 *
+	 * Downloads the avatar image, optimizes it, and stores it in the actors directory.
+	 * Returns the local URL for the saved avatar.
+	 *
+	 * @param string $avatar_url The remote avatar URL.
+	 * @param int    $actor_id   The local actor post ID.
+	 *
+	 * @return string|false The local avatar URL on success, false on failure.
+	 */
+	public static function save_actor_avatar( $avatar_url, $actor_id ) {
```

Comment:
```suggestion
	public static function save_actor_avatar( $actor_id, $avatar_url ) {
```
Let's swap the order to be consistent with other API functions that have the ID first.